lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -158,13 +159,15 @@ class PassiveCalibrationModel:
         Enable hydrodynamic correction.
     distance_to_surface : float, optional
         Distance from bead center to the surface [um].
-        Currently not supported for models without hydrodynamics.
+        None uses the approximation valid for deep in bulk.
```

Comment:
It took me a while to understand this sentence. Not sure I have a good suggestion to rephrase it, but maybe you do. (It's also good to use backticks to distinguish `None` as a keyword.)

Reply:
Yeah, fair point. I'll rephrase it to:
```python